packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -105,11 +99,31 @@ export const getStyles = (props: ICheckboxStyleProps): ICheckboxStyles => {
       ],
       className
     ],
+    input: [
+      {
+        position: 'absolute',
+        background: 'none',
+
+        opacity: 0,
+        selectors: {
+          [`.${IsFocusVisibleClassName} &:focus + label::before`]: {
+            outline: '1px solid ' + theme.palette.neutralSecondary,
+            zIndex: ZIndexes.FocusStyle,

```

Comment:
Confusing to me why this would be needed

Reply:
I mention because z-index values can cause weird side effects when there are things like tooltips or menus covering them.

Reply:
@dzearing  I'm just using the existing logic from the getFocusStyles helper. I would not have known of ZIndexes.FocusStyle otherwise. LMK if you would like me to remove it.

Reply:
Discussed offline and removed.